src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -368,6 +372,15 @@ export const IframeRuntime: React.ForwardRefExoticComponent<IProps> = forwardRef
 
     // Cleanup.
     return () => {
+      // Unregister all components that were proxied through this iframe handler so
+      // we don't leave dangling registered components and so the current component is
+      // silenced if it is being read when the iframe unloads
+      // disable warning about dynamicTextComponentIds as it is not referencing a DOM node
+      // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
You can avoid disabling eslint if you add something like that **above** the cleanup function:

```
const currentDynamicTextComponentIds = dynamicTextComponentIds.current;
```